.vscode/extensions.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
So will you switch to VSCode in the future? I think it'd be cool, if we all have the same IDE^^

Reply:
I'll switch most likely. It just feels much better on the first try :)

Requirement for me right now: Dont't break anything with intellij. Formatting could be a problem.

But I found a plugin for vscode which formats the files according to the eslint rules, which should solve all the problems.